AUTHORING_MANIFESTS.md

```diff
@@ -77,3 +86,135 @@ It is important to test your manifest before submission to ensure it meets the r
+
+### How to make pre-release versions available
+
+If the pre-release versions have different metadata, then just appending the appropriate term to the mainline PackageIdentifier might be enough. The appropriate term would depend on how the developers refer to their early releases. And this new PackageIdentifier would replace the mainline PackageIdentifier in pre-release manifests, manifest names, and manifest paths. 
+
+Some examples: 
+
+Microsoft.OpenSSH: 
+
+- Mainline PackageIdentifier: `Microsoft.OpenSSH`
+- Mainline Repository path: ./manifests/m/Microsoft/OpenSSH
+
+- Beta PackageIdentifier: `Microsoft.OpenSSH.Beta`
+- Beta Repository path: ./manifests/m/Microsoft/OpenSSH/Beta
+
+Microsoft Edge: 
+
+- Mainline PackageIdentifier: `Microsoft.Edge`
+- Mainline Repository path: ./manifests/m/Microsoft/Edge
+- AppsAndFeaturesEntries:
+  - DisplayName: Microsoft Edge
+  - UpgradeCode: '{883C2625-37F7-357F-A0F4-DFAF391B2B9C}'
+
+- Beta PackageIdentifier: `Microsoft.Edge.Beta`
+- Beta Repository path: ./manifests/m/Microsoft/Edge/Beta
+- AppsAndFeaturesEntries:
+  - DisplayName: Microsoft Edge Beta
+  - UpgradeCode: '{55B884A6-908D-3E59-BDCE-5E4BFA64FA7B}'
+
+Use the version number that the pre-release software adds to Apps and features Settings page, or other locations including the Registry, so that the WinGet package manager can provide updates to the pre-release version. 
+
+Note: Pre-release PackageIdentifiers are subordinate to mainline PackageIdentifiers. If the pre-release PackageIdentifier is causing an issue in the mainline, such as causing an upgrade loop by having an identical UpgradeCode, then the mainline should have priority and the pre-release version should be modified or removed.
+
+### Support for release channels (released, beta, alpha)
+
+Notes, ideas, and discussion on how to implement and support release channels [is ongoing](https://github.com/microsoft/winget-cli/issues/147).
+
+## Troubleshooting
+
+###  Display Version Overlap: 
+
+- Can be caused by the PackageVersion differring from the DisplayVersion at any point in the manifest history, and the fix is to add the PackageVersion to every manifest. Another option is to remove the manifests where they differ.
+- Actual log example: `2024-01-31T22:16:48.6603831Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index: [ [5.17.5 (31030), 5.17.5 (31030)]]`
+- DisplayVersion is treated as a range, from the lowest manifest version in the repo to the highest. 
+- This might lead to unexpected matching situations. For example, if the version ranges are 5.17.2 to 5.17.29988, and a PR's version is 5.17.5, then it falls inside that range.
+- In the future, it might be changed from a range, to an array of known manifest DisplayVersions, and require exact matching.
```

Comment:
Talking about future can be confusing. I would rather see "There is ongoing discussion around this behavior in <issue_link>" or no mention at all